src/main/java/org/apache/commons/lang3/text/translate/LookupTranslator.java

```diff
@@ -46,15 +52,19 @@
      *
      * @param lookup CharSequence[][] table of size [*][2]
      */
-    public LookupTranslator(final CharSequence[]... lookup) {
+    @SuppressWarnings({"value:enhancedfor.type.incompatible", "index:argument.type.incompatible"}) /*
+    #1: CharSequence is of the type [*][2] (according to the documentation), hence Charsequence has length 2
```

Comment:
I have already annotated `seq` with `@ArrayLen(2)`

Reply:
Yes, but a warning is issued there because the typechecker can't enforce that annotation. I was suggesting writing an annotation on `lookup` that indicates that its elements are `@ArrayLen(2)`. You've currently annotated it as if its elements are `@MinLen(1)`, instead, for some reason.